src/components/dictionaryConcepts/containers/EditConcept.jsx

```diff
@@ -247,11 +247,16 @@ export class EditConcept extends Component {
   updateEventListener = (event, url) => {
     const { value, name } = event.target;
     const { mappings } = this.state;
+    const source = 'source';
```

Comment:
Shouldn't you use a constant for the above?

Reply:
I have specified the constant in the helperFunction.js component.